server/uno.go

```diff
@@ -181,3 +184,12 @@ func checkForWinner() string {
 	}
 	return ""
 }
+
+func checkForCardInHand(card model.Card, player string) bool {
```

Comment:
@Thomas-Ray-OReilly , these changes will need to be updated to reflect the latest changes from dev.